Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>I wouldn't even bring this one up and instead during the NegBinomial mention that sometimes people say to use a Poisson for count data, but those people are idiots that never worked with real-world data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>